db/store.go

```diff
@@ -738,6 +852,14 @@ func (cs *ChainStore) persist(b *Block) error {
 	// BATCH PUT VALUE
 	cs.st.BatchPut(bhash.Bytes(), hashWriter.Bytes())
 
+	expiredKeys := cs.GetExpiredKeys(b.Header.Height)
```

Comment:
The element of expiredKeys does not contain prefix ”SYS_ExpireKey + height". But BatchDelete need the full key that contain this prefix to delete expired keys.

Reply:
Actually, I should delete two keys, both with prefix and without. Since expired keys contains full key names that should be deleted at specified block.

Reply:
cannot use subscribePayload.Duration (type uint32) as type uint16 in argument to cs.Subscribe